admin/templates/admin/profiles.html

```diff
@@ -0,0 +1,103 @@
+{% extends 'admin/base.html' %}
+
+{% block head %}
+  <title>Modify Profiles - Curling timer</title>
+{% endblock %}
+
+{% block header %}
+  Modify Profiles
+{% endblock %}
+
+{% block content %}
+<div class="card">
+    <h2>Current profiles</h2>
+        <table style="width:100%; text-align: justify;">
```

Comment:
This is SUPER minor and cosmetic only, but it bugged me that the "justify" text align causes the "per" in the "Time per end" column to go way off to the right:
<img width="902" height="323" alt="1" src="https://github.com/user-attachments/assets/6051ed75-0261-45cb-add5-b1d57ce86813" />

Very minor, but my suggestion would be to make it "left" aligned just to make the spacing look a bit better, if you don't want it to be center-aligned.
<img width="869" height="314" alt="2" src="https://github.com/user-attachments/assets/fcc7284c-24c7-4e7b-9cbf-df41321bd73c" />


```suggestion
        <table style="width:100%; text-align: left;">
```